src/main/java/datawave/microservice/authorization/user/ProxiedUserDetails.java

```diff
@@ -24,33 +23,69 @@
 @XmlRootElement
 public class ProxiedUserDetails implements UserDetails {
     private String username;
-    private Set<DatawaveUser> proxiedUsers = new LinkedHashSet<>();
+    private List<DatawaveUser> proxiedUsers = new ArrayList<>();
```

Comment:
Would it ever be problematic to have duplicate proxied users in the list? Technically, we could have duplicate proxied users when using a list. A LinkedHashSet would give you control over the ordering, and handle deduplication (if needed).

Reply:
@jwomeara - made changes